posthog/tasks/insight_query_metadata.py

```diff
@@ -0,0 +1,13 @@
+import structlog
+from celery import shared_task
+
+from posthog.models import Insight
+
+logger = structlog.get_logger(__name__)
+
+
+@shared_task(ignore_result=True)
```

Comment:
Let's put this into a specific background queue so it doesn't bog up other important tasks.

Do we need retry logic here?

Reply:
I've added it to the long_running queue. mainly because we might add more logic to parse HogQL here that would mean it might take longer to process. I think it'll still be around a few seconds then so let me know if another queue (or a separate new one) might make more sense here.

For retry, I think we can add a simple one time retry here just in case the DB is unavailable or there's some other unexpected error, but since I'll be adding a separate sweeping job later I think that'd take care of any missing values.